pandas/core/generic.py

```diff
@@ -9372,68 +9376,80 @@ def to_csv(self, path_or_buf=None, sep=",", na_rep='', float_format=None,
         sep : character, default ','
             Field delimiter for the output file.
         na_rep : string, default ''
-            Missing data representation
+            Missing data representation.
         float_format : string, default None
-            Format string for floating point numbers
+            Format string for floating point numbers.
         columns : sequence, optional
-            Columns to write
+            Columns to write.
         header : boolean or list of string, default True
             Write out the column names. If a list of strings is given it is
-            assumed to be aliases for the column names
+            assumed to be aliases for the column names.
             .. versionchanged:: 0.24.0
                 Previously defaulted to False for Series.
         index : boolean, default True
-            Write row names (index)
+            Write row names (index).
         index_label : string or sequence, or False, default None
             Column label for index column(s) if desired. If None is given, and
             `header` and `index` are True, then the index names are used. A
             sequence should be given if the object uses MultiIndex.  If
             False do not print fields for index names. Use index_label=False
-            for easier importing in R
+            for easier importing in R.
         mode : str
-            Python write mode, default 'w'
+            Python write mode, default 'w'.
         encoding : string, optional
             A string representing the encoding to use in the output file,
             defaults to 'ascii' on Python 2 and 'utf-8' on Python 3.
-        compression : {'infer', 'gzip', 'bz2', 'zip', 'xz', None},
```

Comment:
Think the domain of potential values should remain here - was this causing some type of validation error?

Reply:
In fact the length of the potential values plus the default value span on two lines with the 80 characters limit. This leads to the following error:
`Parameter "compression" description should start with a capital letter`.